article/detorakis-2016.tex

```diff
@@ -295,72 +295,101 @@ \section{Methods}\label{methods}
 along with Numpy (version $1.10.4$), Scipy (version $0.17.0$), Matplotlib
 ($1.5.1$) and Scikit-image (version $0.12.3$). The numerical integration
 has been done using the \emph{ode} method of Scipy \emph{integrate} package.
-Three different methods have been used in this work (\emph{dopri5}, 
-\emph{Adams}, \emph{BDF}, \cite{ascher:1998}). \emph{dopri5} is quite close
-to the one used by the author in the original article (the author has
-numerically integrated the system of the four ODEs by using a fifth-order
-adaptive size Runge-Kutta method). \emph{BDF} and \emph{Adams} provide
-exactly the same numerical results as the first one, but they are
-faster\footnote{The user has the option to choose one of the three methods.
-    Initially,
-    we tested all three methods and since there is no any numerical difference
-    on our results we decided to use \emph{BDF}, since its much faster than
-    \emph{dopri5}.}.
+Three different methods have been tested in this work (\emph{dopri5}, 
+\emph{Adams}, \emph{BDF}, \cite{ascher:1998}). \emph{dopri5} is the closest
+numerical method to the one used by the author in the original article (the
+author has numerically integrated the system of the four ODEs by using a
```

Comment:
"(the author has ...": it is not entirely clear whether "the author" here refers to Wang or to you. Write "Wang" for clarity" or cut it to just "(fifth-order adaptive stepsize Runge-Kutta)".